tests/test_qr_input.py

```diff
@@ -0,0 +1,19 @@
+"""
+Test process
+"""
+
+from modules import qr_input
+
+CAMERA = 1
+
+
+def test_qr_input():
```

Comment:
Change this so that the file is run on its own, rather than through Pytest. A test is not a unit test when it has dependencies.